hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -549,4 +549,8 @@ private OzoneConsts() {
    * tenant.
    */
   public static final int OZONE_MAXIMUM_ACCESS_ID_LENGTH = 100;
+
+  public static final String OM_SNAPSHOT_NAME = "snapshotName";
```

Comment:
This is not used.

Reply:
used here: https://github.com/apache/ozone/blob/d551aa4635e4d33b0080d46d49d8d5b26aead820/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/SnapshotInfo.java#L404